core/src/main/java/org/apache/carbondata/core/constants/CarbonV3DataFormatConstants.java

```diff
@@ -61,24 +61,8 @@
   short NUMBER_OF_COLUMN_TO_READ_IN_IO_MIN = 1;
 
   /**
-   * number of rows per blocklet column page
-   */
-  @CarbonProperty
-  String NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE = "number.of.rows.per.blocklet.column.page";
-
-  /**
    * number of rows per blocklet column page default value
    */
-  String NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT = "32000";
-
-  /**
-   * number of rows per blocklet column page max value
-   */
-  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_MAX = 32000;
-
-  /**
-   * number of rows per blocklet column page min value
-   */
-  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_MIN = 8000;
+  short NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT = 32000;
```

Comment:
I do not think it is configurable, better to remove it from this file

Reply:
this is not configurable. 32000 is the default constant stored as "NUMBER_OF_ROWS_PER_BLOCKLET_COLUMN_PAGE_DEFAULT"